README.MARCJOSEPH.md

```diff
@@ -0,0 +1,31 @@
+# Car Rules 
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
+2.	Passengers are limited to Katrina Wills and family. Any other passengers require permission.
+3.	The car must be kept clean at all times.
+
+
+## Gas Maintenance and Fueling
+maintenance considerations
+
+1.	Gas will be provided only for to and from school; any personal outings are under your responsibility for fuel costs. For personal outings not related to schooling or church, fuel usage must be calculated in advance and paid for by you. Maintenance of the vehicle will be done by you unless someone else damages it or puts it in a position where it could be damaged. Insurance for the car is your responsibility.
```

Comment:
Parents will put $60 gas each week, and car should never be brought home with less than 1/4 of a tank. 

Parents will put $100 maintenance every two months, and car may not be driven unless car meets maintenance standards (create a section that delineates car maintenance standards)

Reply:
Please elaborate on perks

Reply:
update: make it "up to $50 bi-weekly for gas and up to $50 bi-monthly maintenance (to be reviewed as needed)"

Reply:
and put that under "Perks" section

Reply:
and I'm not going to require you calculate fuel in advance.  We can just talk about it and make a pull request for the specific outing.